### PR TITLE
RichText: Style fixes

### PR DESCRIPTION
**Pull request by Mitten-O**
Mostly about indentation and line breaking. As specified in the discussion at #332.

### CODE REVIEW

File: GG/src/RichText/ImageBlock.cpp [hunk 1]
```diff
@@ -15,13 +15,14 @@ namespace GG {
     const std::string ImageBlock::IMAGE_TAG("img");
 
     ImageBlock::ImageBlock(const std::string& image_path, X x, Y y, X w,
-                           GG::Flags<GG::WndFlag> flags)
-    : BlockControl(x, y, w, flags), m_graphic(0)
+                           GG::Flags<GG::WndFlag> flags):
```
Comment (geoffthemedio) [inline GG/src/RichText/ImageBlock.cpp]: space before :

File: GG/src/RichText/ImageBlock.cpp [hunk 2]
```diff
@@ -15,13 +15,14 @@ namespace GG {
...
+        BlockControl(x, y, w, flags), m_graphic(0)
```
Comment (geoffthemedio) [inline GG/src/RichText/ImageBlock.cpp]: each item in initializer list on a new line

File: GG/src/RichText/ImageBlock.cpp [hunk 3]
```diff
@@ -15,13 +15,14 @@ namespace GG {
...
     {
         try {
             boost::shared_ptr<Texture> texture = GetTextureManager().GetTexture(image_path);
-            m_graphic =
-                new StaticGraphic(texture, GRAPHIC_PROPSCALE | GRAPHIC_SHRINKFIT | GRAPHIC_CENTER);
+            m_graphic = new StaticGraphic(
```
Comment (geoffthemedio) [inline GG/src/RichText/ImageBlock.cpp]: why all the linebreaks...? could just move the new ... to the same line as m_graphic =

**Comment by Mitten-O** [comment]
I think I got them all, now?

Warning: rebased.

**Comment by Mitten-O** [comment]
Warning: rebased.

**Comment by geoffthemedio** [comment]
A few minor issues left, but the changes here look good.

**Pull request merged**